pyworkflow/gui/project/viewprotocols_extra.py

```diff
@@ -274,17 +274,20 @@ class ProtocolTreeConfig:
     TAG_PROTOCOL_GROUP = 'protocol_group'
     TAG_PROTOCOL_BETA = 'protocol_beta'
     TAG_PROTOCOL_NEW = 'protocol_new'
+    TAG_PROTOCOL_UPDATED = 'protocol_updated'
     PLUGIN_CONFIG_PROTOCOLS = 'protocols.conf'
 
     @classmethod
-    def getProtocolTag(cls, isInstalled, isBeta=False, isNew=False):
+    def getProtocolTag(cls, isInstalled, isBeta=False, isNew=False, isUpdated=False):
```

Comment:
Why don't we take this chance and pass here the dev status instead of all possible values?

Reply:
there would be more changes needed in searchprotocol...

Reply:
ok, then we leave it. I hope we do not end up with 10 more status. ;-)